lib/gauge_wilson_flow.cu

```diff
@@ -0,0 +1,198 @@
+#include <quda_internal.h>
+#include <tune_quda.h>
+#include <gauge_field.h>
+
+#include <jitify_helper.cuh>
+#include <kernels/gauge_wilson_flow.cuh>
+#include <instantiate.h>
+
+namespace quda {
+
+  template <typename Float, int nColor, QudaReconstructType recon> class GaugeWFlowW1 : TunableVectorYZ
```

Comment:
Similarly to the above comment on using a kernel.  Should be using a single Tunable instance here for the three kernel variants.